google-cloud-firestore/src/main/java/com/google/cloud/firestore/AggregateQuery.java

```diff
@@ -53,7 +53,7 @@ public class AggregateQuery {
     this.query = query;
   }
 
-  /** Returns the query whose aggregations will be calculated by this object. */
+  /** @return the query whose aggregations will be calculated by this object. */
```

Comment:
The sentence starts after the `@return` annotation, so make the first letter a capital. Please do this for other texts as well.

```suggestion
  /** @return The query whose aggregations will be calculated by this object. */
```